src/main/java/io/github/swagger2markup/internal/document/builder/MarkupDocumentBuilder.java

```diff
@@ -225,18 +225,19 @@ protected Type createInlineObjectType(Type type, String name, String uniqueName,
                 
                 MarkupDocBuilder descriptionContent = copyMarkupDocBuilder();
                 String description = defaultString(swaggerMarkupDescription(property.getDescription()));
-                if (isNotBlank(description))
+                if (isNotBlank(descriptionContent.toString())) 
```

Comment:
This first test should test 'description' instead of 'descriptionContent.toString()'.
Other tests below  are OK.